### PR TITLE
[MOB-5314] linting changes to test files and config files

### PR DESCRIPTION
**Pull request by evangreer91**
## 🔹 JIRA Ticket(s) if any

* [MOB-5314](https://iterable.atlassian.net/browse/MOB-5314)

## ✏️ Description

This pull request applies linting changes to the test files and config files.

### CODE REVIEW

File: ts/__tests__/Iterable.spec.ts [hunk 1]
```diff
@@ -130,13 +129,13 @@ test("trackPurchase_params_methodCalled", () => {
...
-  Iterable.logger.log("trackPurchase_paramsWithOptionalFields_methodCalled")
+test('trackPurchase_paramsWithOptionalFields_methodCalled', () => {
+  Iterable.logger.log('trackPurchase_paramsWithOptionalFields_methodCalled')
 
   // GIVEN the following parameters
   const total = 5
-  const items = [new IterableCommerceItem("id", "swordfish", 64, 1, "SKU", "description", "url", "imageUrl", ["sword", "shield"])]
-  const dataFields = {"key": "value"}
+  const items = [new IterableCommerceItem('id', 'swordfish', 64, 1, 'SKU', 'description', 'url', 'imageUrl', ['sword', 'shield'])]
+  const dataFields = { key: 'value' }
```
Comment (Ayyanchira) [inline ts/__tests__/Iterable.spec.ts:138]: key doesnt need " or '?

File: ts/__tests__/Iterable.spec.ts [hunk 2]
```diff
@@ -159,43 +158,43 @@ test("trackEvent_params_methodCalled", () => {
...
+  const campaignId = 1234
+  const templateId = 5678
+  const messageId = 'qwer'
 
   // WHEN Iterable.setAttributionInfo is called with the given attribution info
   Iterable.setAttributionInfo(new IterableAttributionInfo(campaignId, templateId, messageId))
 
   // THEN Iterable.getAttrbutionInfo returns the given attribution info
-  return Iterable.getAttributionInfo().then(attributionInfo => {
+  return await Iterable.getAttributionInfo().then(attributionInfo => {
```
Comment (Ayyanchira) [inline ts/__tests__/Iterable.spec.ts:173]: Wondering how block functions can turn into async via es lint changes. What made it suggest this change?

**Review by Ayyanchira — APPROVED** [review]
Looking great!

File: ts/__tests__/Iterable.spec.ts (hunk 1 above)
Comment (evangreer91) [reply ts/__tests__/Iterable.spec.ts:138]: The linting is following this rule [here](https://typescript-eslint.io/rules/quotes/). We can turn off the rule if necessary.

Comment (Ayyanchira) [reply ts/__tests__/Iterable.spec.ts:138]: This one is good actually...

**Pull request merged**